static/js/components/forms/BulkEnrollmentForm.js

```diff
@@ -114,8 +119,30 @@ export class BulkEnrollmentForm extends React.Component<
     const { successResponseData } = this.state
     const { productMap } = this.props
 
-    const initialProductType = PRODUCT_TYPE_COURSERUN
-    const initialProductId = getFirstKeyAsInt(productMap[initialProductType])
+    const initialProductType = this.state.productType
+
+    const sortedProductIds = Object.keys(productMap[initialProductType]).sort(
```

Comment:
I am not sure about the scalability issue in order to just sort this drop down.

Reply:
I tried to fix it in Python which would not compromise scalability. But **python does not preserve the dictionary order in less than Python 3.6**. **Currently we are using Python 3.5.2 for it**. So it cannot be utilized so **only the frontend option is feasible for now.**

Here is the reference of it.
https://bugs.python.org/issue38414